site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -65,11 +65,11 @@ sudo service fusionauth-search start
 sudo service fusionauth-app start
 ```
 
-With installation complete and services started, let's open up a web browser and navigate to [http://127.0.0.1:9011](http://127.0.0.1:9011) or the web address pointing to your FusionAuth installation. If you're connected over SSH, be sure to first tunnel in to port 9011 so that we can access the server's localhost.
+With installation complete and services started, let's open up a web browser and navigate to [http://127.0.0.1:9011](http://127.0.0.1:9011) or the web address pointing to your FusionAuth installation. If you're connected over SSH, be sure to first tunnel into port 9011 so that we can access the server's localhost.
 
 ## Configuring FusionAuth
 
-The first time you access your new FusionAuth installation, you'll be prompted to [complete the Setup Wizard](/docs/v1/tech/tutorials/setup-wizard). The Setup Wizard will prompt you for your database endpoint, root user credentials, and credentials for the FusionAuth to access the database (you can use existing MySQL credentials here, or FusionAuth will create a new MySQL user). You'll create an administrator account that you'll use every time you login to the FusionAuth UI. After accepting the license and hitting submit you'll need to enter your database Superuser credentials. Then you'll come to a dashboard where you'll be prompted to create an application, a FusionAuth API key, and server email settings. E-mail settings can be found in the FusionAuth UI under **Settings-->Tenants-->Edit Tenant-->Advanced-->SMTP Settings**.
+The first time you access your new FusionAuth installation, you'll be prompted to [complete the Setup Wizard](/docs/v1/tech/tutorials/setup-wizard). The Setup Wizard will prompt you for your database endpoint, root user credentials, and credentials for FusionAuth to access the database (you can use existing MySQL credentials here, or FusionAuth will create a new MySQL user). You'll create an administrator account that you'll use every time you login to the FusionAuth UI. After accepting the license and hitting submit you'll need to enter your database Superuser credentials. Then you'll come to a dashboard where you'll be prompted to create an application, a FusionAuth API key, and server email settings. E-mail settings can be found in the FusionAuth UI under **Settings-->Tenants-->Edit Tenant-->Advanced-->SMTP Settings**.
```

Comment:
>**Settings-->Tenants-->Edit Tenant-->Advanced-->SMTP Settings**.

This is no longer true, it is now at **Tenants-->Edit Tenant-->Email-->SMTP Settings**.

And we use this format in blog posts now:

```
Navigate to "Tenants". Edit your tenant, then choose the "Email" tab. Navigate to the "SMTP settings" section to update your email settings, if needed.
```

Reply:
fixed